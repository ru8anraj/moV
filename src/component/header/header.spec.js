import React from 'react';
import { shallow } from 'enzyme';
import Header from './header.jsx';

describe('<< Header >>', () => {
  it('should render correctly', () => {
    const component = shallow(<Header />);
    expect(component).toMatchSnapshot();
  });
});