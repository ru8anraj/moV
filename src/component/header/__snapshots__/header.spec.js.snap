// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<< Header >> should render correctly 1`] = `
<div
  className="header"
>
  <span
    className="header-text"
  >
    Mo-v
  </span>
</div>
`;
